packages/kokoas-client/src/pages/projInvoice/fieldComponents/selectInvoices/ListItemInvoices.tsx

```diff
@@ -1,29 +1,30 @@
-import { Chip, Stack, Typography } from '@mui/material';
+import { Box, Chip, Stack } from '@mui/material';
 import { IInvoices } from 'types';
-import NumbersIcon from '@mui/icons-material/Numbers';
-import ScheduleIcon from '@mui/icons-material/Schedule';
-import CurrencyYenIcon from '@mui/icons-material/CurrencyYen';
 import { format, parseISO } from 'date-fns';
-import { ReactNode } from 'react';
+import { TInvoiceStatus } from '../../form';
+import { Caption } from 'kokoas-client/src/components';
+import { ButtonContentDataId } from './ButtonContentDataId';
+import { ButtonContentBillingAmount } from './ButtonContentBillingAmount';
 
 
-const TextWithIcon = ({
-  Icon,
-  label,
-}: {
-  Icon: ReactNode,
-  label: string
-}) => {
-  return (
-    <Stack spacing={0} direction={'row'}>
-      {Icon}
-      <Typography>
-        {label}
-      </Typography>
-    </Stack>
-  );
+export type DataIds = {
+  dataId: string,
+  billingAmount: string,
 };
 
+type InvStatusJa = {
+  [key in TInvoiceStatus]: string;
+};
+
+const invoiceStatusJa: InvStatusJa = {
+  'created': '作成済',
+  'sent': '発行済',
+  'voided': '破棄',
+  'completed': '完了',
+  '': '',
+};
```

Comment:
Typescript utilityに便利なRecordがあります。:heart: 

https://www.typescriptlang.org/docs/handbook/utility-types.html

```suggestion

const invoiceStatusJa: Record<InvStatusJa, string> = {
  'created': '作成済',
  'sent': '発行済',
  'voided': '破棄',
  'completed': '完了',
  '': '',
};
```

Reply:
参考URLありがとうございます。
[f75fc1c](https://github.com/Lorenzras/yumecoco-monorepo/pull/154/commits/f75fc1c90687dff8ee860c6e2cf6c69c469671b9)にて、上記内容で修正しました。
ご確認をお願いいたします。